kie-server-parent/kie-server-router/kie-server-router-client/src/main/java/org/kie/server/router/client/KieServerRouterEventListener.java

```diff
@@ -168,8 +170,15 @@ public void afterContainerDeactivated(KieServer kieServer, KieContainerInstance
     protected boolean send(String url, String containerId, String payload, boolean add, boolean retry) {
 
         try {
-            KieServerHttpRequest httpRequest = KieServerHttpRequest.newRequest(url)
-                    .followRedirects(true)
+            KieServerHttpRequest httpRequest = null;
+            // auth will be used if password is provided. user will be the server id
```

Comment:
@elguardian Would it be possible to add optional system property specifying username? And if username is not specified then Kie server id is used?
That would allow customers to specify credentials according their needs.

Reply:
no.

Reply:
@elguardian To be honest, this seems to me like a big overkill. Imagine users having tens of KIE Servers connected to one Smart Router. Then they will have to create a separate account for each kie server just to have the connection secured. Moreover, if they want to change the password, they will have to change all lines in the property file. The idea of @sutaakar is quite good actually. Of course, if they want to have it per kie server, it will be possible. But I imagine that vast majority will use just one or two accounts for many kie servers. It is the same as with kie server connection to controller.

Reply:
public static final String KIE_ROUTER_MANAGEMENT_USERNAME = "org.kie.server.router.management.username";

Overkiliing is having this behavior in here for several reviews and reviewers and change it now